lib/views/pay/pay_view.dart

```diff
@@ -27,10 +30,34 @@ class _PayViewState extends State<PayView> {
   String? createdTime;
   String? expirationTime;
 
-  Future<AppPayInvoice> payInvoice(String boltString) async {
-    final response =
-        await widget.provider.get<AppApi>().payInvoice(invoice: boltString);
-    return response;
+  Future<void> payInvoice(String boltString) async {
+    AppPayInvoice? response;
+    try {
+      response =
```

Comment:
```suggestion
      var response =
```

Reply:
I think we can remove this whole from `var response = await ....` to `await ....`